README.md

```diff
@@ -2,4 +2,5 @@
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
 
 # Artefacts  
-* [PRD](/prd/Juntos_PRD.pdf) 
+* [Product Requirement Document](/prd/Juntos_PRD.pdf) 
```

Comment:
It's preferred to keep the commits in the PR limited to only contain relevant changes

Reply:
I thought you want me to change the name of the PRD. It is supposed to be capital only, so instead, I changed the entire name. You want me to revert it?

Reply:
No need to revert this is more of a FYI comment.

The 'lowercase' comment was for the "filename" i.e. "juntos_system_design.pdf" is preferred over "Juntos_system_design.pdf".

Reply:
oh! I will take care of this moving forward